include/cose/cose_configure.h.in

```diff
@@ -132,17 +151,6 @@
 #define USE_ECDSA_SHA_256
 #define USE_ECDSA_SHA_384
 #define USE_ECDSA_SHA_512
-#if !defined(USE_MBED_TLS)
-//  MBEDTLS currently supports ECDH for X25519 but not EdDSA
-#if OPENSSL_VERSION_NUMBER > 0x10100000L
-// Requires OPEN SSL 1.1.1 to build
-#define USE_EDDSA
-#else
-#pragma message("OPENSSL VERSION IS ")
-#pragma message(OPENSSL_VERISON_NUMBER)
-#endif
-#endif // !defined (USE_MBED_TLS)
-
```

Comment:
Is there a reason for deleting this or have you just not merged?

Reply:
Its above to reduce duplicate code